### ISSUE TITLE
Review from Zaheduzzaman Sarker

### ISSUE THREAD

**Issue by beurdouche** [mlswg/mls-architecture#184]
```
Zaheduzzaman Sarker has entered the following ballot position for
draft-ietf-mls-architecture-10: Discuss

When responding, please keep the subject line intact and reply to all
email addresses included in the To and CC lines. (Feel free to cut this
introductory paragraph, however.)


Please refer to https://www.ietf.org/about/groups/iesg/statements/handling-ballot-positions/ 
for more information about how to handle DISCUSS and COMMENT positions.


The document, along with other ballot positions, can be found here:
https://datatracker.ietf.org/doc/draft-ietf-mls-architecture/



----------------------------------------------------------------------
DISCUSS:
----------------------------------------------------------------------

Thanks for working on this specification.

After reading section 7, I would like to discuss why there is no explicit
recommendation of using secure transport for MLS. This section and subsections
point out various strong opinions to use secure transport. I there is any
particular reason to support secure transport protocol then it should be
mentioned. I kind of feel that the work "transport" here does not really only
refer to Layer 4 transport protocols, needs some clarification.

I also support Roman's discuss that the recommendation on section 7.4.3.2 and
section 7.1.2 are countering each other.


----------------------------------------------------------------------
COMMENT:
----------------------------------------------------------------------

- It took time for me to understand that the recommendation in section 7.1.4
refers to different set of transport protocols than those examples mentioned in
beginning of section 7.1 :-), yes, I did not took the "unidirectional" word
seriously. However, I think some clarification and/or reference to some FEC
scheme for unidirectional transport here would be good.
```

**Comment by rohan-wire** [comment]
Regarding the DISCUSS, a general recommendation to use a reliable, secure transport like TLS or QUIC was merged in PR #183 

Regarding the FEC comment, we believe that anyone working with (ancient) unidirectional transports will be familiar with the art and will not require any reference we might dig up.

Recommend closing now.

**Comment by beurdouche** [comment]
Verified as resolved ! Thank you !
